aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -970,3 +1048,108 @@ resource "aws_elasticache_replication_group" "bar" {
     }
 }`, rInt, rInt, rInt, rInt, rName)
 }
+
+var testAccAWSElasticacheReplicationGroupEnableAtRestEncryptionConfig = fmt.Sprintf(`
```

Comment:
Nitpick: It's a common convention to turn these configs into functions and pass the random strings from the body of the test as argument. It makes it easier to build & compare fields which include these random strings and also makes the config reusable (as it doesn't contain a random string generated once during initiation of the variable).

I know we don't follow that convention everywhere yet, but it's been a common theme of many recent test changes. 😉